core/src/main/resources/error/error-classes.json

```diff
@@ -496,6 +496,11 @@
     ],
     "sqlState" : "42000"
   },
+  "GROUPING_EXPRESSIONS_NOT_FOUND" : {
+    "message" : [
+      "Not found any GROUP BY expressions, and <sqlExpr> is not an aggregate function. Wrap <aggExprs> by windowing function(s) or wrap <sqlExpr> by `first()` (or `first_value()`) if you don't care which value you get."
+    ]
+  },
```

Comment:
@srielau Are you ok with the error class and message format?

Reply:
What is more likely? The user forgot the GROUP BY clause, the user forgot to aggregate the expression, or they meant to use window functions? My money is on forgetting the GROUP BY clause....
How about:
GROUPING_EXPRESSION_WITHOUT_GROUP_BY
The query including grouping expression <groupingExpr> and aggregate expressions <aggExprs> does not include a GROUP BY clause. Add "GROUP BY <groupingExpr>", aggregate <groupingExpr>, or turn the <aggExprs> into window functions using OVER clauses.